JavaScript/Topics/5.Syntax/Conditionals/README.md

```diff
@@ -0,0 +1,221 @@
+# Conditional Statements
+
+Conditional Statements are often used in any programming language to perform different actions based on different conditions.
+
+The conditional statements in JavaScript are:
+
+1. ```if``` statement.
+2. ```if``` - ```else``` statement.
+3. ```else if``` statement.
+4. ```switch``` statement.
+
+## JavaScript ```if``` statement
+
+In this, the part of the code will only execute if the condition given is ```true```.
+
+__Syntax:__
+```js
+if(condition)
+{
+    // code to be executed if the condition satisfies
+}
```

Comment:
```js
if (condition)
{
    ...
}
```
This is NOT the preferred way to have code blocks in JavaScript.
```js
if (condition) {
    ...
}
```

Why so? Because of JavaScript ASI and its flaws, which might misinterpret the earlier form as a JavaScript Object literal. Please modify all code blocks in the file to use the latter.